tests/test_pipelines_automatic_speech_recognition.py

```diff
@@ -42,9 +42,9 @@ def test_torch_small(self):
             tokenizer="facebook/s2t-small-mustc-en-fr-st",
             framework="pt",
         )
-        waveform = np.zeros((34000,))
+        waveform = np.linspace(0, 1, 34000, dtype=np.float32)
```

Comment:
Now that the slight variability due to type-casting is eliminated, this needs to be something specific, because the model becomes unstable (non-reproducible in different environments) with all-zero inputs.